test/provision/k8s_install.sh

```diff
@@ -62,18 +86,18 @@ case $K8S_VERSION in
     "1.9")
         KUBERNETES_CNI_VERSION="0.6.0-00"
         K8S_FULL_VERSION="1.9.7"
-        KUBEADM_SLAVE_OPTIONS="--discovery-token-unsafe-skip-ca-verification"
+        KUBEADM_SLAVE_OPTIONS="--discovery-token-unsafe-skip-ca-verification --ignore-preflight-errors=cri"
```

Comment:
Why are we ignoring preflight errors? It's key that we document why certain configuration options are provided.

Reply:
By default the `kubeadm` is using `/var/run/dockershim.sock` but when we run with containerd the default socket is /var/run/docker.sock, with this change it'll fallback to `docker.sock` or `containerd.sock`

Reply:
Please document this accordingly in the script itself, otherwise people who work on this in the future will not understand why these options are provided.